### DISCUSSION TITLE
[Kavita] Add Magazine library type

### DISCUSSION THREAD

**Discussion by superbonaci** [Kareadita/Kavita#3123] [Ideas]
### Idea Description

When adding a new Library in http://127.0.0.1:5000/admin/dashboard#libraries
there could be also the `Magazine` type.

### Idea Category

Feature Enhancement

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by majora2007** [comment]
Related:
https://github.com/Kareadita/Kavita/discussions/2541

This was actually partially implemented but I haven't had time to go back. 

If anyone is interested in taking it up from me, `feature/magazine`

**Comment by superbonaci** [comment]
Magazines can be in cbr format, or pdf inside rar. If you want example torrents just ask.

**Comment by Camui87** [comment]
Would be awesome if this type could be reimplemented and also would have the option to choose the cover type for always the cover of the latest issue in the library. So that instead like ine the other types where the standard cover is the first issue it would be an option to always use the cover of the last one in the directory
